EssentialCSharp.Web/Areas/Identity/Pages/Account/ForgotPassword.cshtml.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#nullable disable
```

Comment:
for my own edification, is this a .Net web thing to disable nullable on every file?

Reply:
This is for every file this is on.... It's because input model validation with nullability is difficult. Looking into a way to remove it

Reply:
All cases of this should be resolved now